src/local/load.go

```diff
@@ -23,18 +22,25 @@ func (m *Manager) LoadModel(record manifest.Record, target interface{}) (found b
 	}
 
 	// Load values from the meta file
+	metaFilePath := m.Naming().MetaFilePath(record.RelativePath())
 	errPrefix := record.Kind().Name + " metadata"
-	if err := utils.ReadTaggedFields(m.ProjectDir(), record.MetaFilePath(), model.MetaFileTag, errPrefix, target); err != nil {
+	if err := utils.ReadTaggedFields(m.ProjectDir(), metaFilePath, model.MetaFileTag, target, errPrefix); err == nil {
+		record.AddRelatedPath(metaFilePath)
+		m.logger.Debugf(`Loaded "%s"`, metaFilePath)
+	} else {
 		errors.Append(err)
 	}
 
 	// Load config file content
-	errPrefix = record.Kind().Name
 	if configField := utils.GetOneFieldWithTag(model.ConfigFileTag, target); configField != nil {
+		configFilePath := m.Naming().ConfigFilePath(record.RelativePath())
+		errPrefix = record.Kind().Name
 		content := utils.NewOrderedMap()
 		modelValue := reflect.ValueOf(target).Elem()
-		if err := json.ReadFile(m.ProjectDir(), record.ConfigFilePath(), &content, errPrefix); err == nil {
+		if err := json.ReadFile(m.ProjectDir(), configFilePath, &content, errPrefix); err == nil {
 			modelValue.FieldByName(configField.Name).Set(reflect.ValueOf(content))
+			record.AddRelatedPath(configFilePath)
+			m.logger.Debugf(`Loaded "%s"`, configFilePath)
```

Comment:
Pridal som logovanie, ... logovalo sa ukladanie ale nie nacitavanie -> upravene, ak by bol nejaky bug, moze nam to pomoct.